web/crux/templates/dyrector.io.json

```diff
@@ -0,0 +1,399 @@
+            "key": "traefik.http.routers.dyrectorio-crux.service",
+            "value": "dyrectorio-crux"
+          }
+        ],
+        "commands": [
+          {
+            "id": "1",
+            "key": "serve" 
+          }
+        ],
+        "environment": [
+          { 
+            "key": "TZ",
+            "value": "Europe/Budapest"
+          },
+          {
+            "key": "NODE_ENV",
+            "value": "production" 
+          },
+          {
+            "key": "KRATOS_ADMIN_URL",
+            "value": "http://dyrectorio-kratos:4434" 
+          },
+          {
+            "key": "CRUX_AGENT_ADDRESS",
+            "value": "example.com:5001" 
+          },
+          {
+            "key": "CRUX_UI_URL",
+            "value": "https://example.com" 
+          },
+          {
+            "key": "FROM_NAME",
+            "value": "dyrector.io" 
+          },
+          {
+            "key": "FROM_EMAIL",
+            "value": "mail@example.com" 
+          }
+        ],
+        "secrets": [
+          {
+            "key": "DATABASE_URL",
+            "required": true
+          },
+          {
+            "key": "JWT_SECRET",
+            "required": true
+          },
+          {
+            "key": "SMTP_URI",
+            "required": true
+          }
+        ],
+        "networks": ["dyrectorio"],
+        "networkMode": "bridge",
+        "restartPolicy": "unless_stopped",
+        "deploymentStrategy": "recreate",
+        "initContainers": [
+          {
+            "id": "1",
+            "name": "crux-migration",
+            "registryName": "Docker Hub dyrector.io Registry",
+            "image": "crux",
+            "tag": "latest",
+            "useParentConfig": true,
+            "command": [
+              {
+                "id": "1",
+                "key": "migrate" 
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "name": "crux-ui",
+      "registryName": "Docker Hub dyrector.io Registry",
+      "image": "crux-ui",
+      "tag": "latest",
+      "config": {
+        "name": "crux-ui",
+        "expose": "EXPOSE_WITH_TLS",
+        "dockerLabels": [
+          {
+            "id": "1",
+            "key": "traefik.http.routers.dyrectorio-crux-ui.entrypoints",
+            "value": "websecure"
+          },
+          {
+            "id": "2",
+            "key": "traefik.http.services.dyrectorio-crux-ui.loadbalancer.server.port",
+            "value": "3000"
+          },
+          {
+            "id": "3",
+            "key": "traefik.http.routers.dyrectorio-crux-ui.service",
+            "value": "dyrectorio-crux-ui"
+          }
+        ],
+        "environment": [
+          { 
+            "key": "TZ",
+            "value": "Europe/Budapest"
+          },
+          { 
+            "key": "CRUX_API_ADDRESS",
+            "value": "example.com:5001"
+          },
+          { 
+            "key": "DISABLE_RECAPTCHA",
+            "value": "false"
```

Comment:
For easier usability disabling captcha could be a way, otherwise we need decent docs or links into the template that explains how this could be set up. @m8vago?

Reply:
i would rather include the recaptcha keys as envs, and write a disclaimer to the template description

Reply:
Added the keys with the additional information in the description.

Reply:
@nandor-magyar  can we close this?